README.md

```diff
@@ -1 +1,121 @@
-# capstone-project-tesfaye
\ No newline at end of file
+<a name="readme-top"></a>
+
+<div align="center">
+
+  <h3><b>Tesfaye Capstone README </b></h3>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [📝 License](#license)
+
+
+# 📖 [Portfolio mobile project] <a name="about-project"></a>
+
+**[Portfolio mobile project]** is a project that describe my skills and previous work experience.
+
+## 🛠 Built With <a name="built-with">html and css</a>
+
+### Key Features <a name="key-features"></a>
+
+- **[contact form ]**
+- **[projects show cards]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://eyoab01.github.io/tesfaye-capstone-project/)
```

Comment:
- [ ] The live link you have provided is not working. Kindly make sure you provide a correct link so we it is easy for anyone to check your end product.